custom-maven-distribution/create-custom-maven-distribution.sh

```diff
@@ -93,7 +93,7 @@ cp $tmp_dir/$ge_sample_ext_jar $out_dir/$maven_lib_ext
 
 if [ ! -f "$tmp_dir/$ge_sample_ext_xml" ]; then
   echo -e "${yellow}Downloading common gradle-enterprise.xml configuration for CI builds and developers building locally${nc}"
-  ge_sample_ext_xml_download_url=https://raw.githubusercontent.com/gradle/gradle-enterprise-build-config-samples/master/common-gradle-enterprise-xml-maven-configuration/${ge_sample_ext_xml}
+  ge_sample_ext_xml_download_url=https://raw.githubusercontent.com/gradle/develocity-build-config-samples/master/common-develocity-xml-maven-configuration/${ge_sample_ext_xml}
```

Comment:
Nothing to do with your changes, but this link 404s even with how it exists today. Looks like a directory may have been renamed at some point. We should update it.

Reply:
Nice catch!